components/bl-leafletMap-component/src/helpers/coordinates.js

```diff
@@ -0,0 +1,13 @@
+const regEx = /^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?),\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$/;
+
+export function toCoordinates(coords) {
+  if (Array.isArray(coords) && regEx.test(coords.join(','))) {
+    return coords;
+  } else if (typeof coords === 'string' && regEx.test(coords)) {
+    return coords.split(',');
+  } else {
+    console.error('Leaflet Map: not valid coordinates');
+
+    return [40.74, -73.96];
```

Comment:
where do these coords come from?
is it the default value? 
does make sense to use [0,0]?

Reply:
This is the default value. Coordinates 0,0 are somewhere in the ocean, I think this is not correct.

Reply:
can we use the default value in the component properties?